package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "shift-parser",
-  "version": "6.0.1",
+  "version": "7.0.0",
```

Comment:
Leave this out for the moment; we can get the version bump after everything else.

Reply:
@Shilpi3 this isn't resolved; this PR should leave the version as 6.0.1.